### PR TITLE
Ui enhancement update incident list

### PR DESCRIPTION
**Pull request by jgoley**
[PT Chore](https://www.pivotaltracker.com/story/show/146374497)
[Related PT Story](https://www.pivotaltracker.com/story/show/145854209)

### CODE REVIEW

File: client/controllers/incidentReports/incidentSecondaryDetails.coffee [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Template.incidentSecondaryDetails.helpers
+  detailsOpen: ->
+    Template.instance().detailsOpen.get()
+
+  firstLocationName: ->
+    firstLocation = Template.instance().data.locations[0]
+    firstLocation?.countryName or
+      firstLocation?.admin1Name or
+      firstLocation?.admin2Name or
+      firstLocation?.name
```
Comment (jgoley) [inline client/controllers/incidentReports/incidentSecondaryDetails.coffee]: @nathanathan Is there a better way to do this?

Comment (nathanathan) [inline client/controllers/incidentReports/incidentSecondaryDetails.coffee]: Every location has a name so you can always use that. Is there a reason for using the country name instead? I don't think the admin names will be defined if there is no country name, so you could leave those out.

**Comment by nathanathan** [comment]
Looks good but there are a few bugs:
* The dates are incorrect.
* The associated events lost their links.
* I'm seeing a lot of `formatLocation is not defined` errors in the console.

**Comment by nathanathan** [comment]
👍

**Pull request merged**